app/models.py

```diff
@@ -200,7 +200,8 @@ class Agency:
   # An agency which had at least 1 eligible domain.
   def eligible(report_name):
     return db.table('agencies').search(
-      Query()[report_name]['eligible'] > 0
+      (Query()[report_name]['eligible'] > 0) |
+      (Query()[report_name]['pages_count'] > 0)
```

Comment:
Why not have the `pages_count` determination code determine the value for `eligible`?